<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formsupplier</class>
 <widget class="QWidget" name="Formsupplier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nama_supplierLabel">
      <property name="text">
       <string>Nama Supplier</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nama_supplierLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="kode_supplierLabel">
      <property name="text">
       <string>Kode Supplier</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="kode_supplierLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="alamat_suppierLabel">
      <property name="text">
       <string>Alamat Supplier</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="alamat_supplierLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="telp_suppierLabel">
      <property name="text">
       <string>Telp Supplier</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="telp_supplierLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="fax_supplierLabel">
      <property name="text">
       <string>Fax Supplier</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="fax_supplierLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="email_suppierLabel">
      <property name="text">
       <string>Email Supplier</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="email_supplierLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SImpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableSupplier">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>451</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
